rbi/stdlib/matrix.rbi

```diff
@@ -321,8 +319,6 @@ end
 class Matrix
   include ::Enumerable
 
-  Elem = type_member(:out)
```

Comment:
`Enumerable` defines a type_member, so these subclasses of `Enumerable` must also define a type member.

The better solution here would be to write `Elem = type_member {{fixed: T.untyped}}`, which will allow using the classes without providing a type argument.

Reply:
Thanks @jez, pushed that update!